app/controllers/concerns/organization_scopable.rb

```diff
@@ -15,20 +15,17 @@ def set_tenant
   end
 
   def after_sign_in_path_for(resource_or_scope)
-    if resource_or_scope.staff_account&.deactivated?
-      adoptable_pets_path
-    else
+    if allowed_to?(
+      :index?, Pet, namespace: Organizations,
+      context: {organization: Current.organization}
+    )
       pets_path
```

Comment:
We should probably redirect to the dashboard_index_path, now we have a solid grasp on what the dashboard looks like. I think we set it to pets_path before the dashboard index was a thing.

Reply:
Updated the redirect to dashboard now in ada4ef41cc10004f50d5153d23fbbb6c54873383

Reply:
On that note too, I think the dashboard routes could probably be cleaned up. We don't need the full set of RESTful routes like destroy and create for dashboards I am thinking. Not in the scope of this PR but I'll make an issue for it. Probably a good first issue.